packages/components/src/unit-control/styles/unit-control-styles.ts

```diff
@@ -18,16 +18,6 @@ type SelectProps = {
 	selectSize: SelectSize;
 };
 
-export const Root = styled.div`
-	box-sizing: border-box;
```

Comment:
We should probably look into adding `box-sizing: border-box;` back to this component — we have two choices:

1. add it only to `UnitControl` (by adding the rule to the `ValueInput` styled component, as done for the `${ BackdropUI }` rule)
2. add it to [`InputControl`'s `Root` component](https://github.com/WordPress/gutenberg/blob/77b940d669217fc917d246df44dd087cf6345b08/packages/components/src/input-control/styles/input-control-styles.tsx), which means that the style will apply to `InputControl`, `NumberControl` and `UnitControl`

Option 1 is the safer, but option 2 seems the most reasonable to me in order to ensure that those 3 control component have the same `box-sizing` behaviour — but we should be careful and make sure that we won't introduce 

What do folks think?

Reply:
I initially removed the `box-sizing: border-box;` as it had no impact and was already applied on the inner input control field's wrapper. I don't think it hurts to add the reset consistently, though.

Given the input field's wrapper has the reset, the possibility for visual regressions seems pretty limited. I added the style to the input control's `Root` component and reviewed the controls I found that used `InputControl` in both the Storybook and editors. I didn't spot any issues. 

For the sake of being careful, getting an extra set of eyes on this would be appreciated.